### PR TITLE
Refactor builtin runner tests

### PR DESCRIPTION
**Pull request by Juan-M-V**
# Refactor builtin runner tests

## Description

Use macros in the builtin runner tests to reduce verbosity

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: src/vm/runners/builtin_runner.rs [hunk 1]
```diff
@@ -547,27 +546,9 @@ mod tests {
 
     #[test]
     fn deduce_memory_cell_pedersen_for_preset_memory_valid() {
-        let mut memory = Memory::new();
+        let memory = memory![((0, 3), 32), ((0, 4), 72), ((0, 5), 0)]; //Memory::new();
```
Comment (entropidelic) [inline src/vm/runners/builtin_runner.rs:549]: remove the commented code?

**Review by entropidelic — COMMENTED** [review]
Good work, just delete the commented code and we are good to go

File: src/vm/runners/builtin_runner.rs (hunk 1 above)
Comment (Juan-M-V) [reply src/vm/runners/builtin_runner.rs]: siryessir https://github.com/lambdaclass/cairo-rs/pull/407/commits/17370f51df84fe560b323bc6152262fda03b5139

**Review by fmoletta — APPROVED** [review]
Nice!

**Review by pefontana — APPROVED** [review]

**Pull request merged**